civihr_default_theme/scss/pages/_my-leave.scss

```diff
@@ -0,0 +1,10 @@
+.block-civihr-leave-absences {
```

Comment:
But I can't see the block anymore, and in fact (see the screenshot below) the margin-bottom on the button is not applied)